platform/commonUI/general/res/sass/_inspector.scss

```diff
@@ -50,83 +42,194 @@
+            $h: 1.2em;
+            box-sizing: border-box;
+            cursor: pointer;
+            display: inline-block;
+            line-height: $h;
+            position: relative;
+            padding: 2px 4px;
+            .t-object-label {
+                .t-item-icon {
+                    height: $h;
+                    width: 0.7rem;
                 }
             }
-            .form-row {
-                align-items: center;
-                border: none !important;
-                margin-bottom: 0 !important;
-                padding: $interiorMarginSm 0;
-                .label {
-                    min-width: 80px;
-                }
-                input[type='text'],
-                input[type='search'] {
-                    width: 100%;
+            &:hover {
+                background: $colorItemTreeHoverBg;
+                color: $colorItemTreeHoverFg;
+                .icon {
+                    color: $colorItemTreeIconHover;
                 }
             }
         }
+        &:not(.last) .t-object-label .t-title-label:after {
+            color: pushBack($colorInspectorFg, 15%);
+            content: '\e904';
+            display: inline-block;
+            font-family: symbolsfont;
+            font-size: 8px;
+            font-style: normal !important;
+            line-height: inherit;
+            margin-left: $interiorMarginSm;
+            width: 4px;
+        }
     }
 
-    ul li,
-    em.t-inspector-part-header {
-        display: block;
-        position: relative;
+    // Elements pool
+    .holder-elements {
+        .current-elements {
+            position: relative;
+            .tree-item {
+                .t-object-label {
+                    // Elements pool is a flat list, so don't indent items.
+                    font-size: 0.75rem;
+                    left: 0;
+                }
+            }
+        }
     }
+}
 
-    ul li {
-        margin-bottom: $interiorMarginLg;
-    }
+/* Styles for the Inspector pane */
 
-    em.t-inspector-part-header {
-        border-radius: $basicCr;
-        background-color: $colorInspectorSectionHeaderBg;
-        color: $colorInspectorSectionHeaderFg;
+.l-inspect,
+.l-inspect table tr td {
+    font-size: 0.75rem;
+}
+
+.l-inspect {
+    //@extend .abs;
+    //background: $colorInspectorBg;
+    //color: $colorInspectorFg;
+    //line-height: 140%;
+
+    .flex-elem.holder:not(:last-child) {
         margin-bottom: $interiorMargin;
-        padding: floor($formTBPad * .75) $formLRPad;
-        text-transform: uppercase;
     }
 
-    .inspector-properties {
-        &:not(.first) {
-            border-top: 1px solid $colorInspectorSectionHeaderBg;
-        }
-        padding: $interiorMarginSm 0;
-        .label {
-            color: $colorInspectorPropName;
-            text-transform: uppercase;
-        }
-        .value {
-            color: $colorInspectorPropVal;
-            word-break: break-all;
-        }
+    //.pane-header {
```

Comment:
We should delete all the commented styles in this file

Reply:
Missed that, thanks. I'll do a sweep and make sure I didn't miss any other commented styles.

Reply:
Good god I left a lot of dirty laundry laying about. I feel ashamed, but have cleaned up the dorm room.